src/test/TEST_bench/benchmark_spgemm.perf.saved.lassen

```diff
@@ -23,23 +23,23 @@ Device Parcsr Matrix-by-Matrix wall clock time = 0.008427 seconds
 # Output file: benchmark_spgemm.out.12
 Device Parcsr Matrix-by-Matrix wall clock time = 0.011918 seconds
 # Output file: benchmark_spgemm.out.13
-Device Parcsr Matrix-by-Matrix wall clock time = 0.122758 seconds
+Device Parcsr Matrix-by-Matrix wall clock time = 0.027048 seconds
```

Comment:
Wow. this is quite a bit faster.

Reply:
Oh. This is because the problem size is smaller.